integration-test/src/test/java/com/datastrato/gravitino/integration/test/catalog/hive/CatalogHiveIT.java

```diff
@@ -307,15 +304,16 @@ private void checkTableReadWrite(org.apache.hadoop.hive.metastore.api.Table tabl
             .map(Object::toString)
             .collect(Collectors.joining(","));
     if (table.getPartitionKeys().isEmpty()) {
-      sparkSession.sql(String.format(INSERT_WITHOUT_PARTITION_TEMPLATE, dbName, tableName, values));
+      sparkSession.sql(String.format("INSERT INTO %s.%s VALUES (%s)", dbName, tableName, values));
```

Comment:
@zivali Thanks for your contribution.

The `InlineFormatString` is to avoid extracting the format string to a constant. Because it makes it harder to read the `String.format` call. At the same time, we may make a balance. If the constant is used multiple times, we can extract a method for it.  
So I think we can extract a method for these `xxx_TEMPLATE` constants.


![image](https://github.com/datastrato/gravitino/assets/38486782/d64ca7c4-8593-4968-b86c-53aae18173a7)

Reply:
Hi @coolderli, thanks for help catching that! I think you are referring to the `SELECT_ALL_TEMPLATE` that is used twice in this file?
The error-prone plugin seems to be not throwing warnings on those constants that are used multiple times.
I will change the usage
https://github.com/datastrato/gravitino/blob/643d5876e8752a384ce47cfecf7e2c56f8ad2d44/integration-test/src/test/java/com/datastrato/gravitino/integration/test/catalog/hive/CatalogHiveIT.java#L300
into
```
long count = sparkSession.sql(getSelectAllSql(dbName, tableName)).count();

private String getSelectAllSql(String dbName, String tableName) {
    return String.format("SELECT * FROM %s.%s", dbName, tableName);
}
```
and identify other `String.format` usage that error-prone didn't throw a warning on. What do you think? :)

Reply:
@zivali I think we just need to fix those throw warnings. @qqqttt123 What do you think?

Reply:
> @zivali I think we just need to fix those throw warnings. @qqqttt123 What do you think?

Agree with you.

Reply:
I use ./gradlew build to build & test in local environment. The error-prone plugin only show warnings on the modifications in this PR. So I think we don't need further change for this PR?

Reply:
@zivali Sorry, I don't make it clear. I think we can extract such a template.
```
private static final String INSERT_WITHOUT_PARTITION_TEMPLATE = "INSERT INTO %s.%s VALUES (%s)";
```

The `INSERT_WITHOUT_PARTITION_TEMPLATE` is used only once for now. But users may use it in the future. So I think we can extract a method for it although we have only used it once now. What do you think?

Reply:
Oh I see. Thanks for clarifying. Sorry I misunderstood that you were referring to existing multiple-time use cases.
Converting to functions makes sense for reusability. I'll make the changes accordingly :)